src/Microsoft.AspNet.Server.Kestrel/Http/Connection.cs

```diff
@@ -104,9 +146,9 @@ private void OnRead(UvStreamHandle handle, int status, Exception error)
                 }
             }
 
-
             try
             {
+                ExtractIP();
```

Comment:
When the connection is re-instated, the remote port or local NIC chosen may change.